### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by palequezal**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -1,49 +1,38 @@
...
-
 private object AppDependencies {
 
   import play.core.PlayVersion
   import play.sbt.PlayImport._
 
   private val playHealthVersion = "3.9.0-play-26"
-  private val logbackJsonLoggerVersion = "3.1.0"
-  private val govukTemplateVersion = "5.54.0-play-26"
+  private val logbackJsonLoggerVersion = "4.8.0"
```
Comment (alijy) [inline project/AppDependencies.scala:9]: `logbackJsonLoggerVersion` and `playHealthVersion` are not needed anymore.

File: conf/application.conf [hunk 2]
```diff
@@ -183,7 +183,7 @@ assets {
 
 mongodb {
   uri = "mongodb://localhost:27017/"${appName}
-  timeToLiveInSeconds = 2592000
+  timeToLiveInSeconds = 3600 #2592000
```
Comment (alijy) [inline conf/application.conf:186]: remove the commented number

File: conf/application.conf [hunk 3]
```diff
@@ -16,7 +16,7 @@ include "frontend.conf"
 
 
 # Fix for SSLEngine closed already exception
-http-verbs.retries.ssl-engine-closed-already.enabled = true
+http-verbs.retries.ssl-engine-closed-already.enabled = false
```
Comment (alijy) [inline conf/application.conf:19]: Shouldn't the retry mechanism stay enabled?

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -1,49 +1,38 @@
...
+  private val govukTemplateVersion = "5.60.0-play-26"
   private val playUiVersion = "8.11.0-play-26"
```
Comment (alijy) [inline project/AppDependencies.scala:11]: `play-ui` could be bumped up to `1.18.0`

Comment (palequezal) [reply project/AppDependencies.scala:11]: Play ui needs removing as it's no longer used but it's not a straightforward job removing the old assets - needs to be done as a larger piece of td

File: conf/application.conf (hunk 3 above)
Comment (palequezal) [reply conf/application.conf:19]: Forgot about that - I was tinkering on Friday

File: project/AppDependencies.scala (hunk 4 above)
Comment (alijy) [reply project/AppDependencies.scala:9]: :+1:

**Review by alijy — APPROVED** [review]

**Pull request merged**